README.md

```diff
@@ -12,7 +12,7 @@ For installation and usage instructions, see our [documentation web site](https:
 
 ## Building
 
-Golang is required to build the integration. We recommend Golang 1.11 or higher.
+Golang is required to build the integration. We recommend Golang 1.16 or higher.
```

Comment:
could you modify also [line 37](https://github.com/newrelic/nri-nginx/pull/69/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L37) referencing to govendor